wordpress-mobile/WordPress-Android
# [Gutenberg] Correct Color for RichText links in Dark Mode - Testing Build

**Fixes** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2399

**Related PRs:**
`gutenberg` https://github.com/WordPress/gutenberg/pull/23289
`gutenberg-mobile` https://github.com/wordpress-mobile/gutenberg-mobile/pull/2404
`AztecEditor-iOS` https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1295
`WordPress-iOS` TBD

## To test:
Please see the testing and overview in https://github.com/wordpress-mobile/gutenberg-mobile/pull/2404

This is just a version bump to generate a build. 

## PR submission checklist:
